Support/Python/unitybuild/main.py

```diff
@@ -535,7 +543,7 @@ def get_exe_name(platform, exe_base_name):
     exe_name = os.path.join(output_dir, get_exe_name(platform, exe_base_name))
     cmd_env = os.environ.copy()
     cmdline = [
-        get_unity_exe(get_project_unity_version(project_dir), lenient=is_jenkins),
+        get_unity_exe(get_project_unity_version(project_dir), lenient=True),
```

Comment:
ideally, this would be `is_jenkins or is_linux`, but I'm not terribly bothered by this. FWIW, the Jenkins stuff isn't use (our CI is via Github Actions)